runtime/locale/en/status_ailment.hcl

```diff
@@ -1,49 +1,137 @@
 locale {
     status_ailment {
-        blinded {
-            apply = "${name(_1)} ${is(_1)} blinded."
-            heal = "${name(_1)} can see again."
+        message {
```

Comment:
Status ailment's messages are nested now, but reference to the messages in C++ code have not been changed.

`dmgcon` and `healcon` defined in `src/status_ailment.cpp` should be fixed.